src/ipinfo.rs

```diff
@@ -168,6 +195,8 @@ mod tests {
 
     fn get_ipinfo_client() -> IpInfo {
         return IpInfo::new(IpInfoConfig {
+            country_file_path: None,
+            eu_file_path: None,
```

Comment:
Similar here, shouldn't we always default to the `src/*.json` files?

Reply:
Here we are just creating a client for test code. The default code is above. Here you can see what the end user interface will look like.